src/aura/hideEvent/hideEvent.evt

```diff
@@ -0,0 +1 @@
+<aura:event type="APPLICATION" description="Event template" />
```

Comment:
hideEvent is yet another unhappy name. please consider changing in the future, but not needed right now